sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/single_page.hpp

```diff
@@ -17,6 +17,6 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _internal
   {
     virtual ~GetPageResultOptions() = default;
     Azure::Nullable<std::string> NextPageToken;
-    Azure::Nullable<uint32_t> MaxPageResults;
+    Azure::Nullable<int32_t> MaxPageResults;
```

Comment:
Did you check if there were any other places that need fixing? I think I remember seeing a couple when I checked last.

Reply:
I think the others already got fixed :)